pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -2004,3 +2004,27 @@ SELECT ST_LineInterpolatePoint('POLYGON((-1.0 0.0, 0.0 0.0, 0.0 1.0, -1.0 1.0, -
 
 statement error st_lineinterpolatepoints\(\): geometry Polygon should be LineString
 SELECT ST_LineInterpolatePoints('POLYGON((-1.0 0.0, 0.0 0.0, 0.0 1.0, -1.0 1.0, -1.0 0.0))'::geometry, 0.2, false)
+
+query T
+SELECT ST_GEOHASH('POINT(0 0)'::geography,8)
```

Comment:
nit: capitalise these as `ST_GeoHash`, `ST_AsEWKT`.
You may also need to rebase to generate these.